### ISSUE TITLE
some questions about SER and RE task inference

### ISSUE THREAD

**Issue by githublsk** [microsoft/unilm#753]
Hi [wolfshow](https://github.com/wolfshow),
I have a question on the belowing:
I using Chinese document for layoutxlm downstream RE task following below steps:
step1: finetuning a SER model with Chinese document;
step2:finetuning a RE model on SER model trained by step 1;
When I do inference, I should get SER result by SER model trained by step 1; then feed the SER results to RE model trained by step 2, is it right for this inference process? in this situation, to complete the RE task, I should use two models trained by step 1 and step 2?

**Comment by githublsk** [comment]
> @githublsk The RE model is often trained with keys and values with ground truth labels. So, basically, you need two models to finish the SER and RE tasks.

Thank you @[wolfshow](https://github.com/wolfshow), when do inference, should I use SER model to get keys and values, and then use RE model to get the linking relation with  keys and values outputed by SER model?
